cypresse2e/cypress/integration/e2eTests/offerHelp/offerHelpQuestion.spec.js

```diff
@@ -76,8 +76,8 @@ describe('Fight Pandemics Offer Help Questionnaire', () => {
             offerHelpQuestion.getLocationInputInfo().contains(locationInputInfo);
         });
 
-        it('Offer Help Question Two Page - Share My Location Icon and Text is visible and clickable', () => {
-            var shareMyLocationText = "Share My Location";
+        it('Offer Help Question Two Page - Share my Location Icon and Text is visible and clickable', () => {
+            var shareMyLocationText = "Share my Location";
```

Comment:
'My' should stay capitalized.

Reply:
I think as well 'My' should be capitalized, as in every situation if an app asking you to give location would show up like that.